rero_mef/agents/mef/views.py

```diff
@@ -44,3 +44,17 @@ def redirect_item(pid):
             'invenio_records_rest.mef_item', pid_value=pid, **request.args),
         code=308
     )
+
+
+api_blueprint_latest = Blueprint(
+    'api_agents_mef_latest',
+    __name__,
+    url_prefix='/agents/mef'
+)
+
+
+@api_blueprint_latest.route('/latest/<pid_type>:<pid>')
+def get_latest(pid_type, pid):
+    """Get latest MEF for pid and type."""
+    from rero_mef.agents.mef.api import AgentMefRecord
```

Comment:
Is this import can't be move to the top of the file ? I don't understand where a circular import could be occurred.